backend/app/models.py

```diff
@@ -4,3 +4,32 @@
 from werkzeug.security import generate_password_hash, check_password_hash
 
 # create model classes here
+
+# Many-to-Many relationships requires helper table
+# According to the flask documentation
+contacts = db.Table(
+    "contacts",
+    db.Column("contact_id", db.Integer, db.ForeignKey("contact.id"), primary_key=True),
+    db.Column("project_id", db.Integer, db.ForeignKey("project.id"), primary_key=True),
+)
+# Project Model
+
+
+class Project(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
```

Comment:
We want to use UUID since basic integer id is not the secure – sorry I mentioned this in the issue for the endpoints but not for the models. 
Something like
 `id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4, unique=True, nullable=False)` https://pypi.org/project/Flask-UUID/